clients/githubrepo/graphql.go

```diff
@@ -89,6 +90,21 @@ type graphqlData struct {
 										}
 									}
 								} `graphql:"reviews(last: $reviewsToAnalyze)"`
+								Commits struct {
+									Nodes []struct {
+										Commit struct {
+											CheckSuites struct {
+												Nodes []struct {
+													App struct {
+														Slug githubv4.String
+													}
+													Conclusion githubv4.CheckConclusionState
+													Status     githubv4.CheckStatusState
+												}
+											} `graphql:"checkSuites(first: $checksToAnalyze)"`
+										}
+									}
+								} `graphql:"commits(last:1)"`
```

Comment:
Nit: can consider returning the merge commit here and comparing against `headRefOid` and log differences. Can be used in debugging later.

Reply:
Nvm this - we've the other log (comparing this against old `listCheckRuns`) that should be enough